lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -799,23 +799,31 @@ def add_auth_network_tokenization(xml, payment_method, options)
 
         brand = card_brand(payment_method).to_sym
 
+        # stored_credential_overrides is not documented on the gateway guide in docs
+        # I think the easiest way to solve for apple pay recurring is to create a new field within the hash
+        # commerce_indicator can be passed in as 'internet' but is assigned that value in an earlier method so will be overwritten by the card brand if there is not something done within this method
+        commerce_indicator = 'internet' if options.dig(:stored_credential_overrides, :type) == 'apple_pay'
```

Comment:
Are you envisioning the merchant passing in this GSF or adding when we build the params in the gateway class?

Reply:
That was a thought, but this was more of a means to an end to see if I could get a passing remote test. I'm open to other options.